pkg/cache/cache.go

```diff
@@ -418,6 +427,8 @@ func (rc *RemoteCache) update(ctx context.Context, orgDesc, newDesc, cacheDesc *
 			Platform:  platform,
 		})
 	}
+	// append cache version annotation
+	cacheIndex.Annotations[LayerAnnotationCacheVersion] = rc.version
```

Comment:
Should add the annotation to manifest instead of index?

Reply:
ok